sample/src/main/java/com/twilio/verify/sample/viewmodel/ChallengesViewModel.kt

```diff
@@ -30,10 +31,9 @@ class ChallengesViewModel(private val twilioVerifyAdapter: TwilioVerifyAdapter)
 
   fun loadChallenges(factorSid: String) {
     twilioVerifyAdapter.getAllChallenges(
-      ChallengeListPayload(factorSid, PAGE_SIZE),
+      ChallengeListPayload(factorSid, PAGE_SIZE, order = Desc),
```

Comment:
Should the default it be Ascending or Descending ?

https://github.com/twilio/twilio-verify-ios/blob/b19e2824168a877b00a9c720bc189580fb23e600/TwilioVerifySDK/TwilioVerify/Sources/Models/ChallengeListPayload.swift#L45

Reply:
To keep the previous ordering for customers already integrated and as backend default value is asc, we should use asc